client/payment-details/summary/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -4341,7 +4341,7 @@ exports[`PaymentDetailsSummary renders partially refunded information for a char
 </div>
 `;
 
-exports[`PaymentDetailsSummary renders the Tap to Pay channel from metadata with ios COTS_DEVICE 1`] = `
```

Comment:
Took me a little bit to understand here that the two snapshots got inverted 😅 I kept thinking "but the test file didn't change (much)!"